public/script/report.js

```diff
@@ -0,0 +1,18 @@
+var locationButton = document.getElementById('getLocationButton');
+
+function getLocation() {
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(showPosition);
+    } else {
+        x.innerHTML = "Geolocation is not supported by this browser.";
+    }
+}
+
+function showPosition(position) {
+    console.log(
+    "Latitude: " + position.coords.latitude +
+    "Longitude: " + position.coords.longitude
+  )
+}
+
+locationButton.addEventListener('click', getLocation)
```

Comment:
why separate getLocation out? you could even get away with not having any variables, e.g.
```
document.getElementById('getLocationButton').addEventListener('click', function() {
    ...
});
```